e2e-tests/limits/compare/statefulset_no-limits-pxc-increased-oc.yml

```diff
@@ -248,7 +248,7 @@ spec:
       initContainers:
         - command:
             - /pxc-init-entrypoint.sh
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent
```

Comment:
why do we change this value?

Reply:
With this PR, we set `ImagePullPolicy` to the same value for both init containers and regular ones. This change is to fix failing tests.